quantum/rgblight_breathe_table.h

```diff
@@ -1,466 +1,115 @@
 #ifndef RGBLIGHT_EFFECT_BREATHE_TABLE
```

Comment:
```suggestion
#pragma once
```

Reply:
Actually, I think this is intentional: 
https://github.com/qmk/qmk_firmware/blob/e318d2ca1554a80e1a407bf263d5c55da33afbe2/quantum/rgblight.c#L857-L858

Reply:
Fair enough; deleted the suggestions removing this define (but kept the pragma once).